### DISCUSSION TITLE
Markup editor appears on pressing Enter

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#377] [Help]
I must have clicked or somehow changed the Freeplane options. But suddenly, every time I want to create a new node, FP creates it and immediately opens an Edit Markdown dialog for me to write in. This is instead of writing directly inline in the node.  It is different from the node's text editor. There's no toolbar - just an open pane that says Edit Markdown.

![image](https://user-images.githubusercontent.com/8197883/168959783-cf4e977d-cf58-4c6f-a33a-507951f9e7f0.png)

What setting did I inadvertently change?

Thanks.

~ Bal

**Comment by macmarrum** [comment]
Yep, I ran into this some time ago too 😁 

Command Search `Ctrl+F1` helped me to find the culprit

![CommandSearch_markdown](https://user-images.githubusercontent.com/7197131/168961849-9d9d22da-4424-48bc-8c02-82c910f7d056.png)

`Preferences->Plugins->Markdown->Edit Markdown in-line`

![Edit_Markdown_in-line](https://user-images.githubusercontent.com/7197131/168961525-cd05a272-133b-4992-b446-4aead17ff124.png)

Alternatively, the Markdown format might have been set (unintentionally) on your (Default) style. Switching back to Standard would also fix the unwanted Markdown Editor 😄 

![Style_CoreText_Markdown](https://user-images.githubusercontent.com/7197131/168960931-27a22ddd-6c43-49ac-a42a-b2911be1dd74.png)

**Reply by dpolivaev** [reply]
Amazing explanation, thanks a lot

**Comment by wordmuse** [comment]
Perfect - thank you! I wonder how I did that inadvertently? Anyway - I'm
glad I did. Now I know about Command Search. All these years, and I never
noticed that before!

As Dimitry said, amazing explanation.

Thank you!

~ Bal
